src/styles/components/card.scss

```diff
@@ -0,0 +1,85 @@
+@import "./utilits/colors.scss";
+@import "./utilits/mixins.scss";
+
+.cards-container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  padding: 200px 0 72px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+    padding: 120px 0 72px;
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+    padding: 50px 0 72px;
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  max-width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
What about `transition` for this scale and for all other animations on the page

Reply:
Why didn't you add `transition`, it will improve your work several times